rllib/tests/run_regression_tests.py

```diff
@@ -97,6 +97,9 @@
         if args.framework in ["tf2", "tfe"]:
             exp["config"]["eager_tracing"] = True
 
+        if int(os.environ.get("RLLIB_NUM_GPUS", "0")):
+            exp["config"]["num_gpus"] = 1
```

Comment:
why do you want to force this? we should respect users' input.

Reply:
oh understand why you need to check the env var this way, I am more suggesting the following:
```
num_gpus = int(os.environ.get("RLLIB_NUM_GPUS", "0"))
if num_gpus:
    exp["config"]["num_gpus"] = num_gpus
```

I was only curious about the hardcoded number 1 there. I don't see why we want to force the test to only use 1 GPU.

Reply:
Oh silly me I'm dumb. Yeah you're right